animeone/android/app/src/main/res/layout/activity_webview.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <WebView
+        android:id="@+id/webView"
+        android:visibility="invisible"
```

Comment:
There might be some issues here because the WebView is invisible, but it can still work and fix the issue.